Content.Server/_Stalker/AdvancedSpawner/AdvancedRandomSpawnerSystem.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Maths;
+using Content.Server._Stalker.AdvancedSpawner;
+using Content.Server.TrashDetector.Components;
+using Robust.Shared.Prototypes;
+using Content.Shared.Popups;
+using System.Numerics;
+using Robust.Shared.Log;
+
+namespace Content.Server._Stalker.AdvancedSpawner
+{
+    public sealed class AdvancedRandomSpawnerSystem : EntitySystem
+    {
+        [Dependency] private readonly IRobustRandom _random = default!;
+        [Dependency] private readonly SharedPopupSystem _popupSystem = default!;
+        private ISawmill _sawmill = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            _sawmill = IoCManager.Resolve<ILogManager>().GetSawmill("AdvancedSpawner");
```

Comment:
```suggestion
            _sawmill = Logger.GetSawmill("advanced_spawner");
```
логгеры лучше называть в снейк кейсе, хз почему, но оффы делают так, не выделяемся, не отсвечиваем